package.json

```diff
@@ -43,5 +44,11 @@
     "ts-node": "^9.1.0",
     "typescript": "^4.1.2",
     "yargs": "^16.1.1"
-  }
+  },
+  "files": [
+    "/build",
+    "/deployments",
+    "/lib",
```

Comment:
Whats the lib folder?

Reply:
It's where the index.js and index.d.ts go

Reply:
I see, it was added to the `tsconfig` as the `outDir`